Rakefile

```diff
@@ -148,7 +148,7 @@ namespace :release do
     say "* [ ] Tests (rake spec) are passing, manual verifications complete."
     say "* [ ] Email is composed and ready to send to braintree-sdk-announce@googlegroups.com"
 
-    abort(1) unless ask "Ready to release? "
+    abort(1) unless ask "Ready to release? Press any key to continue. "
```

Comment:
Is it a ton more work to have the `Y` and `N` keys programmed to continue or cancel here?

Reply:
Probably not a ton, but it would be some work. I'd say let's leave this as a future improvement 🙂

Reply:
@scannillo Are you good with this being merged?

Reply:
YES!